<launch>
  <node name="trajectory_republisher_ra" pkg="sr_robot_commander" type="trajectory_republisher.py" output="screen">
    <remap from="input" to="/ra_trajectory_controller/command_old_stamp"/>
    <remap from="output" to="/ra_trajectory_controller/command"/>
  </node>
  <node name="trajectory_republisher_rh" pkg="sr_robot_commander" type="trajectory_republisher.py" output="screen">
    <remap from="input" to="/rh_trajectory_controller/command_old_stamp"/>
    <remap from="output" to="/rh_trajectory_controller/command"/>
  </node>
  <node name="trajectory_republisher_wr" pkg="sr_robot_commander" type="trajectory_republisher.py" output="screen">
    <remap from="input" to="/rh_wr_trajectory_controller/command_old_stamp"/>
    <remap from="output" to="/rh_wr_trajectory_controller/command"/>
  </node>
  <!-- <node name="trajectory_republisher_la" pkg="sr_robot_commander" type="trajectory_republisher.py" output="screen">
    <remap from="input" to="/la_trajectory_controller/command_old_stamp"/>
    <remap from="output" to="/la_trajectory_controller/command"/>
  </node>
  <node name="trajectory_republisher_lh" pkg="sr_robot_commander" type="trajectory_republisher.py" output="screen">
    <remap from="input" to="/lh_trajectory_controller/command_old_stamp"/>
    <remap from="output" to="/lh_trajectory_controller/command"/>
  </node>
  <node name="trajectory_republisher_wl" pkg="sr_robot_commander" type="trajectory_republisher.py" output="screen">
    <remap from="input" to="/lh_wr_trajectory_controller/command_old_stamp"/>
    <remap from="output" to="/lh_wr_trajectory_controller/command"/>
  </node> -->
</launch>
